extensions/spark/kyuubi-spark-authz/pom.xml

```diff
@@ -261,6 +261,19 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Not a good experience to introduce spark sql engine module here

Reply:
I didn't find the general module of spark-sql-engine, can you provide ideas?

Reply:
I should write a ranger-based OperationManager, thanks!

Reply:
In order to avoid too much repetitive code and support more subsequent extensions, I think it is necessary to write the kyuubi-spark-sql-common module to store abstract interfaces and tool classes.

Reply:
we can format arguments to sql command, then run with spark.sql("command here"). with the result it produces, which is filtered ahead by auth module, we call catalog APIs to get detailed information

Reply:
OK, I will adjust according to this idea.

Reply:
If a large number of tables are queried, the performance loss of "SHOW COLUMNS" will be severe.

Reply:
this case is very rare, most meta APIs are called lazily, that is, when getColums is called, the `catalog/schema/table` part is specific

Reply:
This situation is not uncommon. If we regard kyuubi as a database, does the application generally obtain table information through jdbc meta?

Reply:
sorry, I resubmitted a pr (https://github.com/apache/incubator-kyuubi/pull/2648), the kyuubi code update frequency is quite high, praise!